core/src/main/scala/org/http4s/Uri.scala

```diff
@@ -302,7 +193,7 @@ object Uri extends UriPlatform {
     *
     * @see [[https://tools.ietf.org/html/rfc3986#section-3.1 RFC 3986, Section 3.1, Scheme]]
     */
-  final class Scheme private (val value: String) extends Ordered[Scheme] {
+  final class Scheme private[http4s] (val value: String) extends Ordered[Scheme] {
```

Comment:
I made a comment that got eaten: I think this can be constrained to `Uri` because it's an inner type and be closer to the original.